devicetypes/smartthings/testing/simulated-color-control.src/simulated-color-control.groovy

```diff
@@ -1,6 +1,8 @@
 metadata {
 	definition (name: "Simulated Color Control", namespace: "smartthings/testing", author: "SmartThings") {
     	capability "Color Control"
+		capability "Sensor"
```

Comment:
- Any Device that sends / reports Events / Attribute values (i.e., reports a key State ... not counting something secondary like "Battery"), is a Sensor.

So... If the Device "reports" if something else changes it's color, then it is acting as a Sensor. (I'm checking the code now to see if this is true).

---------
**BTW:** For the purpose of SmartTiles V6 SmartApp (and many similar SmartApps), it actually doesn't matter if Sensor and/or Actuator are properly used, as long as one and/or the other Capability is claimed for every single Device. Every DTH must be one and/or the other; unless it is a very, very weird DTH -- I'd love to know of any such example.

Our SmartApp just uses Sensor / Actuator as master input filters and then query the selected Devices for their detailed *specific* Capabilities in order to know what services to provide for them.